### DISCUSSION TITLE
GitHub Discussions vs ServiceStack.net Forum ?

### DISCUSSION THREAD

**Discussion by techdragon** [ServiceStack/Discuss#35] [Q&A] [answered]
It would be good to get some clarification on the different purposes of the GitHub discussions here, and the Discourse forum over at https://forums.servicestack.net/ since there is some conceptual overlap in their purposes. Also since there doesn't appear to be any mention of the other location at each of them and newer users might not know to look for at the other location for information so it might be good to link between them if they are both to continue going forward.

**Comment by mythz** [comment, marked as answer]
The Customer Forums is only for Customers whilst anyone can ask questions here, we've also moved feature requests from UserVoice to here at https://servicestack.net/ideas where it provides nicer collaboration and visibility.

Basically you can use both for general support, use GitHub Discussions for Feature Requests and stuff like Show and tell (i.e. what cool stuff you've built with ServiceStack) or Polls which have built-in support.

**Reply by techdragon** [reply]
Thanks for the clarification!
